### ISSUE TITLE
Generic build instructions

### ISSUE THREAD

**Issue by origintopleft** [wagoodman/dive#491]
**What would you like to be added**:
Generic "how to build" instructions that don't involve distro-specific tools such as apt or dnf, preferably in an INSTALL.md file in the root.

**Why is this needed**:
I intend to maintain an ebuild for Gentoo in a personal overlay, but can't even figure out how to build manually. I am not coming from a Go background and don't know what the standard `./configure && make` equivalent is. `go build`, as suggested by Golang docs, throws up because the `dive` directory already exists, and I'm unsure how to respond to this.

**Additional context**:
Would also come in handy for users of other distros looking to package this program.

**Comment by mark2185** [comment]
There is a `Makefile` but I'm not sure what target to invoke since the one run on `CI` doesn't seem to work locally.

Anyway, to answer this:
> and I'm unsure how to respond to this

`go build -o dive` will create a `dive` binary in the `./dive/` folder.

**Comment by ChaosEngine** [comment]
ebuild has landed some time in bugs.gentoo.org: https://bugs.gentoo.org/828478

**Comment by wagoodman** [comment]
Agreed, there should be at least a DEVELOPING.md and/or CONTRIBUTING.md files that specify how to get started working in this repo.

**Comment by Bsd-us** [comment]
really looking forward for this, thank you for your work
